frontends/web/src/api/account.ts

```diff
@@ -22,7 +22,7 @@ export type CoinCode = 'btc' | 'tbtc' | 'ltc' | 'tltc' | 'eth' | 'teth' | 'reth'
 
 export type AccountCode = string;
 
-export type Fiat = 'AUD' | 'BRL' | 'BTC' | 'CAD' | 'CHF' | 'CNY' | 'EUR' | 'GBP' | 'HKD' | 'ILS' | 'JPY' | 'KRW' | 'RUB' | 'SGD' | 'USD';
+export type Fiat = 'AUD' | 'BRL' | 'BTC' | 'CAD' | 'CHF' | 'CNY' | 'EUR' | 'GBP' | 'HKD' | 'ILS' | 'NOK' | 'JPY' | 'KRW' | 'RUB' | 'SGD' | 'USD';
```

Comment:
Please sort alphabetically after 'KRW' 

(note: this is not the order in which it is displayed in the UI)

Reply:
Fixed